Sorry: TabError: inconsistent use of tabs and spaces in indentation (inference.py, line 92)
